Runigram.java

```diff
@@ -37,16 +43,27 @@ public static Color[][] read(String fileName) {
 		in.readInt();
 		// Creates the image array
 		Color[][] image = new Color[numRows][numCols];
+		for (int i = 0; i < numRows; i++) 
+		{
+            for (int j = 0; j < numCols; j++) 
+			{
+                int red = in.readInt();
+                int green = in.readInt();
+                int blue = in.readInt();
+                image[i][j] = new Color(red, green, blue);
+            }
+        }
+        return image;

```

Comment:
Indent proper. It's important for the readability!

Also it would be better to use 'row', 'col' variable names instead of i and j